sector-builder/src/disk_backed_storage.rs

```diff
@@ -130,6 +161,96 @@ impl DiskManager {
             })
             .map(|_| access)
     }
+
+    fn convert_sector_id_to_access_name(&self, sector_id: SectorId) -> Result<String, SectorManagerErr> {
+        let seg_id = (sector_id >> 32) as u32;
```

Comment:
This changeset makes modifications to `DiskManager` which allow it (from a type perspective) to be configured to use `SectorAccessProto::Ip`. My read of this code tells me that segment id is only meaningful in the context of `SectorAccessProto::Default`. Is that true? If that is the case, I suggest restructuring the code to make that more clear.

Reply:
No. I mean that DiskManager could be configured to use SectorAccessProto::Ip, or other protos defined in the future. I agree that the DiskManager is to provide a disk access for sectors, or more precisely, file-system access. however, as a filesystem, sector_access could be accessed the same way as local disk, even the real location could be somewhere else, e.g. it's in a folder which is mounted to a network vis NFS, or a local filesystem via iSCSI, or, even a cloud disk. 
Since from the higher level at which go-filecoin is build, there is no difference, all data can be access via file-system calls, the DiskManager could definitely serve it. However, as a miner, we want to distinguish them by segment_id, and it is also good to use the access name we'd like to define. So, why do not we provide this kind of flexibility, and limit it to original one only.

Reply:
I am set this conversation as resolved. Please reopen it if you have more comments or questions.